### PR TITLE
Django 2.0 Support

### PR DESCRIPTION
**Pull request by ArtemBernatskyy**
Added support for Django 2.0, slight changes in PEP 8

### CODE REVIEW

**Comment by ArtemBernatskyy** [comment]
Ok, we will fix coverage and code style

**Comment by millerdev** [comment]
@ArtemBernatskyy thanks for the work on this. Unfortunately we will not be able to merge it for some time because the project we use it in is still on Python 2. We are working toward Python 3, but do not expect to be there anytime soon.

**Comment by ArtemBernatskyy** [comment]
@millerdev can we use different python versions for different Djangos. For example starting from `Django 2` we will enforce to use `Python 3` ? What do you think about it ?

**Comment by ArtemBernatskyy** [comment]
@millerdev i mean that when person `pip install django-prbac` then it receives different versions based on the current user's python version

**Comment by millerdev** [comment]
@ArtemBernatskyy what is your motivation for doing this? In our experience the `six` library provides a reasonable way to support both Python 2 and 3 with a single codebase, and many projects out there do this. TBH, I'm not excited about maintaining two separate branches of the code, one for Django 1.x/Python 2 and one for Django 2/Python 3.

**Comment by emord** [comment]
@ArtemBernatskyy I think you can use the matrix option here: https://docs.travis-ci.com/user/customizing-the-build/#Build-Matrix to remove any combinations of django 2.x and python 2 which should allow for testing both

**Comment by ArtemBernatskyy** [comment]
@emord , thx. But what about pip releases ? Can we allow both versions to exist at the same time ?

File: django_prbac/admin.py [hunk 1]
```diff
@@ -57,25 +60,16 @@ def parameters__csv(self, instance):
         'description',
     ]
 
+
 class GrantAdmin(admin.ModelAdmin):
 
     model = Grant
 
-    def edit_link(self, instance):
```
Comment (ArtemBernatskyy) [inline django_prbac/admin.py]: We don't need this (why this was present in the original code )

**Comment by ArtemBernatskyy** [comment]
@millerdev Check it now, i've reverted support for Python 2

**Comment by millerdev** [comment]
@ArtemBernatskyy looking good. Please separate out the style cleanup (wrapping long lines, removing unused code, etc.) into a separate commit from the changes required for Django 2.0.

File: django_prbac/admin.py (hunk 1 above)
Comment (millerdev) [reply django_prbac/admin.py]: I think fine to remove it as long as all tests still pass.

**Comment by ArtemBernatskyy** [comment]
@millerdev done, also rebased incorporating previous changes. Can you also rebuild the docs ?
Thx

**Comment by ArtemBernatskyy** [comment]
@millerdev done, also rebased incorporating previous changes. Can you also rebuild the docs ?
Thx

File: django_prbac/admin.py (hunk 1 above)
Comment (millerdev) [reply django_prbac/admin.py]: Oh, this appears to be part of the admin interface. I think it should not be removed.

File: django_prbac/decorators.py [hunk 2]
```diff
@@ -5,8 +5,8 @@
 
 # Local Imports
 from django.http import Http404
-from django_prbac.exceptions import PermissionDenied
 from django_prbac.utils import has_privilege
+from django_prbac.exceptions import PermissionDenied
```
Comment (millerdev) [inline django_prbac/decorators.py]: The convention is to order imports alphabetically by module name. Why was this changed?

File: setup.py [hunk 3]
```diff
@@ -19,7 +21,7 @@ def get_version():
     packages=find_packages(),
     zip_safe=False,
     install_requires=[
-        'django>=1.8,<2.0',
+        'django>=1.8,<2.1',
```
Comment (millerdev) [inline setup.py]: This doesn't seem like a PEP 8 or redundancy code change. Probably should be in its own commit.

File: django_prbac/admin.py (hunk 1 above)
Comment (ArtemBernatskyy) [reply django_prbac/admin.py]: Yeh, but it's redundant because Django admin allows the same functionality right from box

File: setup.py (hunk 3 above)
Comment (ArtemBernatskyy) [reply setup.py]: fixed

**Comment by ArtemBernatskyy** [comment]
Closing because it's open source, in that way we don't care about your custom settings (115 characters)
I'll better spend my time developing it on my own

**Comment by millerdev** [comment]
@ArtemBernatskyy do you mind if I merge just the first commit? Django 2.0 Support ?

File: django_prbac/admin.py (hunk 1 above)
Comment (millerdev) [reply django_prbac/admin.py]: Ah, I didn't realize that. In that case, fine to remove.

**Comment by ArtemBernatskyy** [comment]
Sure @millerdev , do what u want.

**Comment by millerdev** [comment]
@ArtemBernatskyy thank you for the contribution. Sorry the code review had so many iterations.
